src/db-adapters/Mongoose/MongooseAdapter.js

```diff
@@ -614,4 +596,29 @@ export default class MongooseAdapter {
 
     return words.join(" ");
   }
+
+  static getIdQueryType(idOrIds) {
```

Comment:
Also, last but not least, please run `make lint` and clean up any errors/warnings that result. We gotta make sure the style etc is consistent :)